.env.example

```diff
@@ -28,4 +28,7 @@ STRIPE_SECRET_KEY=sk_test_51IravfFqipIA40A3FOW6EzlXlJiXjL9V0FXKfb9n7cxh25Ww9QMA9
 # To run in single-campaign mode, set the following environment vars
 VUE_APP_CAMPAIGN_MODE = "single" # or "default" for normaal operation
 VUE_APP_FEATURED_CAMPAIGN = "5" # campaign's id from Postgres
-VUE_APP_LETTER_TEMPLATE = "" # the template associated with the featured campaign
\ No newline at end of file
+VUE_APP_LETTER_TEMPLATE = "" # the template associated with the featured campaign
+
+# Cicero configuration
+CICERO_API_URL=http://localhost:3030
```

Comment:
Do we need to have another process running on a separate port? Check out my other comments below about this.

Reply:
I'm now realizing this it NOT needed because of the way the `json-server` mock routes are injected, so I'll remove it. 

However, I still think the URL for the Cicero API should be defined in the environment like this(that's just good practice in general).